qcelemental/molparse/to_string.py

```diff
@@ -127,7 +128,11 @@ def to_dict(self) -> Dict:
         nat = len(atoms)
 
         first_line = """{} {}""".format(str(nat), umap.get(units.lower(), units.lower()))
-        smol = [first_line.rstrip(), name]
+        smol = [first_line.rstrip()]
+        if dtype == 'xyz':
+            smol.append(name)
+        else:
+            smol.append(f"{molrec['molecular_charge']} {molrec['molecular_multiplicity']} {name}")
```

Comment:
chg mult name isn't a violation of "xyz" format. and "xyz" is already violated by units in the first line, so there's nothing but a choice stopping you from writing the extended seconda line for both xyz and xyz+. up to you.

Reply:
Agreed.

Reply:
In order to make this change, I had to update some tests. Could you please take a look and make sure the new test reference results are correct? I don't know all of the atom symbol annotations used.